tests/PKSim.Tests/Infrastructure/DomainFactoryForSpecs.cs

```diff
@@ -175,6 +175,14 @@ public static Population CreateDefaultPopulation(Individual individual)
       {
          var populationFactory = IoC.Resolve<IRandomPopulationFactory>();
          var populationSettings = IoC.Resolve<IIndividualToPopulationSettingsMapper>().MapFrom(individual);
+         //Non Age dependent species. We have to make sure that we set correct weight so that the algorithm can randomize target weight
+         if (!individual.IsAgeDependent)
```

Comment:
@Yuri05 I am not sure how it ever worked before. The values coming from the DB were always unset (0 to infinity) and as such the weight could not be calculated
Anyways that should work now

Reply:
Life is full of miracles :)